cime_config/tests.py

```diff
@@ -249,6 +249,7 @@
             "SMS_D_Ld1.T62_oQU240wLI.GMPAS-IAF-PISMF.mpaso-impl_top_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-harmonic_mean_drag",
             "SMS_D_Ld1.T62_oQU240.GMPAS-IAF.mpaso-upwind_advection",
+            "SMS_P512_D_Ld5.T62_oEC60to30v3wLI_ais20.MPAS_LISIO_TEST.mpaso-ocn_glcshelf",
```

Comment:
@jonbob, should we leave off the `P512` and let the infrastructure handle it? I think @matthewhoffman was finding that it ran on 15 Chrysalis nodes by default, which seemed to me like a lot.

Reply:
I think we should leave off the P512 and add an entry to the config_pes_tests.xml instead, if necessary. P512 is awkward if it gets run on multiple machines, since the pes/node is variable

Reply:
@jonbob , what's the procedure for updating config_pes_tests.xml?  Would it be better up update config_pes.xml?  I've always been a little uncertain about how to specify compset/grid combinations properly in those files.